content/docs/reference/pg-extensions.md

```diff
@@ -29,15 +22,22 @@ For information about using the Neon SQL Editor, see [Query with Neon's SQL Edit
 | hstore                   | [1.8](https://www.postgresql.org/docs/14/hstore.html)           | [1.8](https://www.postgresql.org/docs/15/hstore.html)            |                                                                                                                    |
+| hypopg                   | [1.3.1](https://hypopg.readthedocs.io/en/rel1_stable/)          | [1.3.1](https://hypopg.readthedocs.io/en/rel1_stable/)           |                                                                                                                    |
+| insert_username (spi)    | [1.0](https://www.postgresql.org/docs/current/contrib-spi.html) | [1.0](https://www.postgresql.org/docs/current/contrib-spi.html)  |                                                                                                                    |
 | intagg                   | [1.1](https://www.postgresql.org/docs/14/intagg.html)           | [1.1](https://www.postgresql.org/docs/15/intagg.html)            |                                                                                                                    |
 | intarray                 | [1.5](https://www.postgresql.org/docs/14/intarray.html)         | [1.5](https://www.postgresql.org/docs/15/intarray.html)          |                                                                                                                    |
 | isn                      | [1.2](https://www.postgresql.org/docs/14/isn.html)              | [1.2](https://www.postgresql.org/docs/15/isn.html)               |                                                                                                                    |
 | lo                       | [1.1](https://www.postgresql.org/docs/14/lo.html)               | [1.1](https://www.postgresql.org/docs/15/lo.html)                |                                                                                                                    |
 | ltree                    | [1.2](https://www.postgresql.org/docs/14/ltree.html)            | [1.2](https://www.postgresql.org/docs/15/ltree.html)             |                                                                                                                    |
+| moddattime (spi)         | [1.0](https://www.postgresql.org/docs/current/contrib-spi.html) | [1.0](https://www.postgresql.org/docs/current/contrib-spi.html)  |                                                                                                                    |
+| pg_graphql               | [1.1.0](https://github.com/supabase/pg_graphql)                 | [1.1.0](https://github.com/supabase/pg_graphql)                  |                                                                                                                    |
```

Comment:
Duplication. Should be pg_jsonschema and pg_graphql

Reply:
Thank you. Fixed.